### PR TITLE
Fix Zig setup step in CI

### PR DESCRIPTION
**Pull request by hronro**
Fix the broken CI of building aarch64 binaries.

And I've tested it in my forked repo: https://github.com/hronro/grpc-web/actions/runs/2451424475

### CODE REVIEW

File: .github/workflows/make-plugin-arm.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@ jobs:
         submodules: 'recursive'
     - name: Setup Zig
       run: |
-        mkdir /zig
-        curl 'https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz' | tar xJ --strip-components=1 --directory=/zig
-        ln -s /zig/zig /usr/bin/zig
+        sudo mkdir /zig
+        curl 'https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz' | sudo tar xJ --strip-components=1 --directory=/zig
+        sudo ln -s /zig/zig /usr/bin/zig
```
Comment (sampajano) [inline .github/workflows/make-plugin-arm.yml:22]: Thanks for the fix!

Curious does this *have* to happen under root directory? If not, would sudo still be necessary?

Comment (hronro) [reply .github/workflows/make-plugin-arm.yml:22]: It's about installing for all users (need root permission) or installing for the current user (don't need permission). I think both of them are OK in CI.

Comment (sampajano) [reply .github/workflows/make-plugin-arm.yml:22]: ahh right.. If you don't mind, i wonder if you could install it for the current user? (in general i'd want to minimize root access unless strictly necessary). Thanks!

Comment (hronro) [reply .github/workflows/make-plugin-arm.yml]: Sure.

**Review by sampajano — APPROVED** [review]
thanks so much for the change!! :)

File: .github/workflows/make-plugin-arm.yml (hunk 1 above)
Comment (hronro) [reply .github/workflows/make-plugin-arm.yml]: lol do you have any simpler solutions?

**Pull request merged**